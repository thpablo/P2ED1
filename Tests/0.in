15 15
c
***************
*   *   *     *
*** * * * *** *
*M* * *     * *
* * *** ***** *
* *   * *     *
* *** * * *****
*     * * *   *
* ***** * * * *
* *     *   * *
* *** ******* *
*   *     *   *
*** ******* * *
*           *  
***************