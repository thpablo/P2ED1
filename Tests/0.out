27
4, 1
5, 1
6, 1
7, 1
8, 1
9, 1
10, 1
11, 1
11, 2
11, 3
12, 3
13, 3
13, 4
13, 5
13, 6
13, 7
13, 8
13, 9
13, 10
13, 11
12, 11
11, 11
11, 12
11, 13
12, 13
13, 13
13, 14
